src/javascript/expression.rs

```diff
@@ -128,6 +130,35 @@ impl<'module> Generator<'module> {
             ValueConstructorVariant::Record { .. } if constructor.type_.is_nil() => {
                 Ok("undefined".to_doc())
             }
+            ValueConstructorVariant::Record { name, arity: 0, .. } => {
+                let record_type = name.to_doc().surround("\"", "\"");
+                let record_head = (RECORD_KEY.to_doc(), Some(record_type));
+                Ok(wrap_object(std::iter::once(record_head)))
+            }
+            ValueConstructorVariant::Record {
+                name,
+                arity,
+                field_map,
+                ..
+            } => {
+                let vars = (0..*arity)
+                    .into_iter()
```

Comment:
Redundant, ranges are already Iterators

Reply:
The human clippy strikes! :P 

Maybe we should have clippy on CI?